src/MembershipsSDKAdapter.js

```diff
@@ -38,6 +44,32 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
     this.membershipSubjects = {};
   }
 
+  /**
+   * Gets members from a room
```

Comment:
@lalli-flores As the listening can only be done on all room memberships (not just the "current" room), we'll have to filter the received events based on roomId. We'll be basically looking for `created` and `deleted` events (maybe `updated` events as well). Also, for simplicity and to preserve the backend order, we were thinking of just reloading the members of a room after such events, instead of trying to amend the member list by pushing the new member at the end of the previously loaded list or filtering out the deleted one. We'd be interesting in hearing your thoughts on this.

Reply:
> @lalli-flores As the listening can only be done on all room memberships (not just the "current" room), we'll have to filter the received events based on roomId. We'll be basically looking for `created` and `deleted` events (maybe `updated` events as well). 
Yes, that's the way to do it. No need to look at `updated`, that is for the read receipts mostly, which we are not implementing at this point.

> Also, for simplicity and to preserve the backend order, we were thinking of just reloading the members of a room after such events, instead of trying to amend the member list by pushing the new member at the end of the previously loaded list or filtering out the deleted one. We'd be interesting in hearing your thoughts on this.

It's ok to send a brand new array from the SDK adapter, not completely sure I like the idea of reloading the whole list in UI each time though. That's a UI issue though, which we should be addressed separately.
As for what the list should look like, please remember that the final list should have the current user at the top, following an alphabetical list of the rest of the members.
Please, [see the Jira for full requirements on this feature](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-154240)